registries/_format/date.md

```diff
@@ -19,3 +20,15 @@ The `{{page.slug}}` format represents a date as defined by full-date - [RFC3339]
 
```

Comment:
RFC link is broken.  https://www.rfc-editor.org/rfc/rfc3339#section-5.6

Reply:
fixed